testsuite/tests/typing-layouts/allow_illegal_crossing.ml

```diff
@@ -12,21 +12,35 @@ type a
 type b : value mod portable = { a : int; b : int }
 [%%expect {|
 type a
-type b : value mod portable = { a : int; b : int; }
+type b = { a : int; b : int; }
 |}]
```

Comment:
This test is no longer testing the thing it was meant to test. `b` is meant to have a type that is normally inferred to not be portable. So I think the `int`s should be replaced with a type that doesn't cross portability.

The same is true for most of the tests in this file. But maybe this isn't worth the effort of fixing, because:
1. We plan to remove illegal crossing very soon, and I believe @glittershark plans to delete this file rather than incorporating it into the trust-me annotation tests.
2. I see that you've added some tests where the inferred type does not cross.

Reply:
Yes good point. I thought about this. MAybe I should have changed existing tests? But I think we still have good enough coverage here, so I think I won't do this. If you're ok with this, please Resolve.